api/apps/articles/model/save.coffee

```diff
@@ -62,11 +62,13 @@ getDescription = (article) =>
   text
 
 removeStopWords = (title) ->
-  title = title.replace(/[.,\/#!$%\^&\*;:{}=\—_`’~()]/g," ")
-  newTitle = _.difference(title.split(' '), stopWords.stopWords).join(' ')
-  return newTitle
+  title = title.replace(/[0-9.\/#!$%\^\*;{}=_`’~()]/g,"")
```

Comment:
Hmm why do we want to move numbers? Is this something that was discussed separately? I think this would affect drafts that don't have titles. In those cases we use dashed dates.